velox/vector/tests/VectorEstimateFlatSizeTest.cpp

```diff
@@ -63,7 +63,7 @@ bool boolAt(vector_size_t row) {
 }
 
 StringView shortStringAt(vector_size_t row) {
-  return StringView(std::string(row % 3, '.'));
+  return &".."[row % 3];
```

Comment:
nit: maybe add a quick one-liner comment to explain what's going on here. It took me a minute.